build-native.sh

```diff
@@ -7,17 +7,24 @@ REPO_ROOT="$SCRIPT_ROOT"
 SOURCE_ROOT="$REPO_ROOT/src"
 BINARY_ROOT="$REPO_ROOT/bin"
 OBJECT_ROOT="$REPO_ROOT/obj"
+SUBMODULE_ROOT="$REPO_ROOT/src/libuv"
 
 BUILD_TYPE=Debug
 CMAKE_BUILD_TYPE=Debug
+BUILD_ARCH=x64
```

Comment:
Can we rename this to `TARGET_ARCH`  I think it will make more sense if/when we want to do cross compilation.